revert/store_all_for_multiple_choice_fields.psql

```diff
@@ -0,0 +1,15 @@
+-- Revert afya-tek:store_all_for_multiple_choice_fields from pg
+
+BEGIN;
+
+-- DROP all views created with adjustments
+DROP VIEW IF EXISTS afyatek_final.child_addo_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.child_home_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.anc_addo_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.anc_home_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.adolescent_addo_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.adolescent_home_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.pnc_addo_visit_final CASCADE;
+DROP VIEW IF EXISTS afyatek_final.pnc_home_visit_final CASCADE;
```

Comment:
Is this the accurate negation of this migration?  The queries here should take back the database deterministically to the exact state it was in before your applied this migration.

Reply:
Revert has been updated to return views to original state rather than drop.